aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,76 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## June 14-18, 2021
+
+- Released grommet-icons v4.6.0! Check out the [Release Notes](https://github.com/grommet/grommet-icons/releases/tag/v4.6.0).
+- Added [download links for MetricHPEXS](/foundation/typography#fonts-for-offline-usage) for offline usage
```

Comment:
Opinion: for consistency and since our bullets read as sentences, recommend concluding each bullet point with a period.

```suggestion
- Added [download links for MetricHPEXS](/foundation/typography#fonts-for-offline-usage) for offline usage.
```

Reply:
```suggestion
- Added [download links for MetricHPEXS](/foundation/typography#fonts-for-offline-usage) font files for offline usage.
```